tasks/strings/strings.js

```diff
@@ -9,9 +9,18 @@
  * console.log(reverseString(123)) // 'This is not a string!'
  */
 function reverseString(str) {
-
+  if (typeof str === "string") {
+    let reversed = "";
+    for (let i = str.length; i >= 0; i--) {
+      reversed += str.charAt(i);
+    }
+    return reversed;
```

Comment:
You should use methods of Array and String here.

Hints: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split